src/cloud/architecture/pro-develop-deploy-workflow.md

```diff
@@ -17,7 +17,9 @@ The following graphic demonstrates the Pro develop and deploy workflow, which us
 
 ## Development workflow {#develop}
 
-The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create up to four additional branches for developing your custom code, extensions, and third party integrations. This allows for a maximum of five _active_ branches deployed to Platform-as-a-Service (PaaS) containers.
+The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create one additional, environment branch. This allows for up to two active branches deployed to Platform-as-a-Service (PaaS) containers.(PaaS) containers.
```

Comment:
```suggestion
The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create one additional environment branch. This allows for up to two active branches deployed to Platform-as-a-Service (PaaS) containers.(PaaS) containers.
```

Reply:
Should this not mention that its one additional active environment? You can have as many inactive environments as you want. Or is this just complicating the docs?